src/components/molecules/Card/Card.js

```diff
@@ -2,76 +2,52 @@ import Title from "~/components/atoms/Title/Title";
+  const remainder = CardsData?.length % 3;
+
   return (
-    <div className="flex flex-wrap mb-4 text-center items-center w-[100%] justify-center mt-5">
-      {CardsData?.map(({ route, name, text, subtitle, id }) => (    
-          <div 
-            className="flex flex-col text-center items-center card-content" // New class for the card content div
-            style={{ 
-              marginBottom: '30px',
-              marginTop: '4px',
-              borderRadius: '0px',
-              margin: '20px',
-              // height: '450px',
-              // boxShadow: '0px 0px 15px 0px rgba(0,0,0,0.75)',
-              // backgroundColor: 'rgba(0, 0, 0, 0.5)', // Add a semi-transparent background color to make text more readable
-              position: 'relative', // Add position relative to the card content div
-            }}
-            key={id}
-          >
-            <div 
-            style={{
-              position: 'absolute', top: 20, zIndex: 1 }}>
-            <Text
-              Style="subtitle"
-              Text={subtitle}
-              TextClassProps="text-white"
-            />
-              </div>
-              <div>
+    <div className="grid grid-cols-2 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 gap-4 mb-4 text-center items-center justify-center mt-5 w-full lg:w-[80rem] xl:w-[80rem] mx-[10rem] lg:mx-[5rem]">
+      {CardsData?.map(({ route, name, text, subtitle, id }, index) => (
+        <div 
+          className={`flex flex-col items-center space-y-4 p-4 w-full ${remainder === 1 && index === CardsData.length - 1 ? 'lg:col-start-2 xl:col-start-2' : ''}`}
+          key={id}
+        >
+          <div className="relative w-full">
+            <div className="absolute top-4 lg:top-5 xl:top-7 z-10 w-full">
+              <Text
+                Style="subtitleCompact"
+                Text={subtitle}
+                TextClassProps="text-white font-semibold text-xs sm:text-xs lg:text-lg xl:text-xl sm:leading-tight sm:mb-1"
+              />
+            </div>
             <ImageAtomLocal
               imagesize="375x450"
               border=""
               src={route}
               alt="Image card"
-              ImageStyleProps="mt-3"
+              ImageStyleProps="mt-3 w-full"
             />
-            
-            {/* <div style={{ position: 'absolute', bottom: 0, right: 0, top: 0, left: 0, width: '100%', height: '%', backgroundColor: 'rgba(0, 0, 0, 0.3)' }}></div> */}
-             </div>
-
-            <div 
-              className="inset-0 bg-black bg-opacity-50 overflow-hidden" 
-              id="inner"
-              style={{
-                position: 'absolute',
-                top: 300,
-                left: 0,
-                right: 0,
-                bottom: 0,
-                paddingBottom: 0 
-              }}
+            <div
+              className="absolute inset-x-0 bottom-0 bg-black bg-opacity-50 p-1 top-[54%] sm:top-[54%] lg:top-[70%] xl:top-[70%]"
```

Comment:
`lg:top-[70%]` should also apply to `xl` breakpoint. `xl:top-[70%]` becomes redundant.